### ISSUE TITLE
Ore cluster finder missing in newest 1.12.2 version

### ISSUE THREAD

**Issue by GamerShowG** [pWn3d1337/Techguns2#296]
Hi, wanted to ask about the ore cluster finder, since it is very difficult to find ore clusters now without the finder, and wanted to ask if it missing is intentional or a bug/forgotten item? Thanks in advance.

**Comment by pWn3d1337** [comment]
1.12 does no longer spawn ore clusters naturally, they are part of structures that spawn in the world

**Comment by GamerShowG** [comment]
Right, thanks, only realized this after finding a couple cluster mines and bug nests
